build_projects/dotnet-cli-build/CompileTargets.cs

```diff
@@ -418,71 +222,27 @@ private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli d
             }
 
             // cleanup compilers project output we don't need
-            DeleteMainPublishOutput(outputDir, "compilers");
+            PublishMutationUtilties.CleanPublishOutput(sdkOutputDirectory, "compilers");
             File.Delete(compilersDeps);
-            File.Delete(compilersRuntimeConfig);
```

Comment:
Should CleanPublishOutput also remove the .deps file so the above line can go away as well?